a10_octavia/controller/worker/tasks/health_monitor_tasks.py

```diff
@@ -49,20 +51,34 @@ def execute(self, listeners, health_mon, vthunder):
                                             health_mon.rise_threshold, method=method,
                                             port=listeners[0].protocol_port, url=url,
                                             expect_code=expect_code, axapi_args=args)
-            LOG.debug("Health Monitor created successfully: %s", health_mon.id)
-        except Exception as e:
-            LOG.exception("Failed to create Health Monitor: %s", str(e))
-            raise
+            LOG.debug("Successfully created health monitor: %s", health_mon.id)
+
+        except (acos_errors.ACOSException, ConnectionError) as e:
+            LOG.exception("Failed to create health monitor: %s", health_mon.id)
+            raise e
 
         try:
             self.axapi_client.slb.service_group.update(health_mon.pool_id,
                                                        health_monitor=health_mon.id,
                                                        health_check_disable=0)
-            LOG.debug("Health Monitor %s is associated to pool %s successfully.",
+            LOG.debug("Successfully associated health monitor %s to pool %s",
                       health_mon.id, health_mon.pool_id)
+        except (acos_errors.ACOSException, ConnectionError) as e:
+            LOG.exception(
+                "Failed to associate health monitor %s to pool %s",
+                health_mon.id, health_mon.pool_id)
+            raise e
+
+    @axapi_client_decorator
+    def revert(self, listeners, health_mon, vthunder, *args, **kwargs):
+        try:
+            self.axapi_client.slb.hm.delete(health_mon.id[:5])
```

Comment:
Please edit this to ` self.axapi_client.slb.hm.delete(health_mon.id)`

Reply:
Have added requested change